### PR TITLE
[W6.2b][W13-3]ChelseyOng

### PR DESCRIPTION
**Pull request by chel-seyy**
Ready for review

### CODE REVIEW

File: src/seedu/addressbook/commands/Command.java [hunk 1]
```diff
@@ -70,4 +70,6 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    public abstract boolean isMutating();
```
Comment (Sri-vatsa) [inline src/seedu/addressbook/commands/Command.java:74]: Having a default implementation of `isMutating()` as an abstract method is great. Good job!! 👍

File: src/seedu/addressbook/commands/Command.java [hunk 2]
```diff
@@ -70,4 +70,6 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
```
Comment (Sri-vatsa) [inline src/seedu/addressbook/commands/Command.java:73]: Don't forget to write a header comment for `isMutating()`. If not, users of the method as well as those who override this method will not know how exactly the method behaves.

**Review by Sri-vatsa — APPROVED** [review]
Please note that going forward you need to update the tests as well when you make any behavior changes.

Once reviewed, you may close the PR :)

**Comment by chel-seyy** [comment]
Noted

**Pull request closed without merging**